deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -252,12 +252,13 @@ def update_watchdir(self, watchdir_id):
                 # Skip directories
                 continue
             else:
-                ext = os.path.splitext(filename)[1]
+                ext = os.path.splitext(filename)[1].lower()
                 if ext == ".torrent":
                     magnet = False
                 elif ext == ".magnet":
                     magnet = True
                 else:
+                    log.warning("File checked for auto-loading is invalid: %s", filename)
```

Comment:
The reason I want to add some sore of logging here is because I spent at least an hour trying to figure out why the auto-add setting wasn't paying any attention to my specified folder. in reality, it was paying close attention to that folder, but silently failing. I see your concern, though.

An alternative that I can think of right now would be to add a debug message to the extension check to just say something like "evaluating file for auto-add." 

I still stand behind adding some form of increased visibility.